sc/shared/src/test/scala/sigmastate/utxo/BasicOpsSpecification.scala

```diff
@@ -765,4 +767,25 @@ class BasicOpsSpecification extends CompilerTestingCommons
     test("subst", env, ext, hostScript, null)
   }
 
+  property("Box.getReg") {
+    def getRegTest(): Assertion = {
+      test("Box.getReg", env, ext,
+        """{
+          |   val x = SELF
+          |   x.getReg[Long](0).get == SELF.value &&
+          |   x.getReg[Coll[(Coll[Byte], Long)]](2).get == SELF.tokens &&
+          |   x.getReg[Int](9).isEmpty
+          |}""".stripMargin,
+        null
+      )
+    }
+
+    if (VersionContext.current.isV6SoftForkActivated) {
+      getRegTest()
+    } else {
+      an[Exception] should be thrownBy getRegTest()
+    }
+
+  }
```

Comment:
Having test coverage in LSV6 this test doesn't bring much value.
The negative case also better test in LSV6 (there are examples in LSV5 how to do this)

Reply:
I disagree, with the help of a test in BasicOpsSpecification compiler bug in https://github.com/ScorexFoundation/sigmastate-interpreter/pull/984 was found and fixed , and LSV6 did not help with that

Reply:
I'm not against unit test, but BasicOpsSpecification is not "unit"-test-like suite, it tests both the compiler and the interpreter, so basically serving the purpose of LSVx. 
I'm also ok with leaving this test here, but it looks like duplication to me.

Reply:
Yes, BOS is checking serialization / interpreter / compiler, but approach is different (more complex programs, in particular), thus it is possible to catch bugs with it where LSV is missing them (see example above) , and also works for providing useful examples of new features usage (see https://github.com/ergoplatform/ergo/releases/tag/v6.0.0-alpha0 , which is suggesting to find examples in this suite).